### DISCUSSION TITLE
ghcr.io: Transferring project to Organization doesn't redirect pull requests

### DISCUSSION THREAD

**Discussion by gaby** [community/community#57785] [Packages]
### Select Topic Area

Bug

### Body

We recently migrated a project from an individual account to an organization through the Project Settings page. The transfer left the existing Docker Images in the individual account without a repo attached to them instead of transferring them to the new Org. Now when users do a pull they get an image that's several weeks old unless they change the image name.

- Why isn't gchr.io redirecting pull requests when a project gets transferred to an Organization?
- Why isn't the transfer project functionality moving current packages to the new location?

**Comment by DeepanshKhurana** [comment]
This is what has been preventing me from being able to move my open-source repositories into my organisation. Such a huge, huge limitation. What does transfer even mean if it is a partial transfer?
